W-21917743 apim relaunch may 11 jh (#355)
mulesoft/docs-exchange: modules/ROOT/pages/adding-azure-api-management-scanner.adoc

= Adding a Scanner for Azure API Management
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> Add a scanner to discover, import, and sync agents from Azure API Management into Exchange. Then you can govern the agents and consume them in other applications.

Reviewer: @JennyHajee 
```suggestion
Add a scanner to discover, import, and sync services from Azure API Management into Exchange. Once imported, these services are available for governance, policy enforcement, and exposure through your developer portal.
```

> Add a scanner to discover, import, and sync services from Azure API Management into Anypoint Exchange. Once imported, you can apply governance, enforce policies, and publish them to your developer portal.

Reviewer: ```suggestion
Add a scanner to discover, import, and sync APIs from Azure API Management into Anypoint Exchange. Once imported, you can apply governance rulesets and publish them to your developer portal.
```

== Before You Begin
Before adding the scanner, verify that you have the permission, context, and these Amazon API Gateway credentials:

* Exchange Administrator permission 
* API Management Service Reader Role permission
* Tenant ID
* Client ID
* Client Secret
* Subscription ID
* Resource Group
* Service Name

== Add a Scanner for Azure API Management
. Verify that you are in the business group where you want to add the scanner.
. From the sidebar in Exchange, click *Scanners*.
. Enter a name for the scanner.
. From *Scanner Run Configuration*, complete these fields or options:
+
[%header,cols="1,4"]
|===
|*Field/Option* |*Value*
|*Run Schedule* |Select a frequency and local time.
|*Sync Review* |Select an option: Auto-resolve or Ask to review.
|===

. From *Connection Configuration*, complete these fields:
+
[%header,cols="1,4"]
|===
|*Field* |*Value*
|*Provider* |Select *Microsoft*.
|*Platform* |Select *Azure API Management*.

> |*Service Type* |Select *REST APIs*.

Reviewer: @JennyHajee 
```suggestion
|*Service Type* |*REST APIs* will be preselected.
```

> |*Authentication Method* |Select *OAuth*.

Reviewer: @JennyHajee 
```suggestion
|*Authentication Method* |*OAuth* will be preselected.
```

> |*Tenant ID* |Enter the access key ID.

Reviewer: @JennyHajee 
```suggestion
|*Tenant ID* |Enter the tenant ID.
```

> |*Client ID* |Enter the secret access key.

Reviewer: @JennyHajee 
```suggestion
|*Client ID* |Enter the client id.
```
